### PR TITLE
Update raw3-5

### PR DESCRIPTION
**Pull request by destogl**
I would like to merge updated paramters of raw3-5

### CODE REVIEW

**Comment by destogl** [comment]
Extends #399

**Comment by floweisshardt** [comment]
CR: OK

File: cob_bringup/robots/raw3-5.xml [hunk 1]
```diff
@@ -23,17 +23,35 @@
 		</include>
 
 		<!-- startup hardware -->
-		<include file="$(find cob_bringup)/drivers/laser_lms1xx.launch" >
+		<include file="$(find cob_bringup)/drivers/sick_lms1xx.launch" >
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="name" value="base_laser_front" />
 		</include>
-		<include file="$(find cob_bringup)/drivers/laser_lms1xx.launch" >
+		<include file="$(find cob_bringup)/drivers/sick_lms1xx.launch" >
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-5.xml:30]: the renaming makes sense...but are we sure, it is used nowhere else?

File: cob_bringup/tools/base_collision_observer.launch [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+  <node pkg="cob_collision_velocity_filter" type="collision_velocity_filter" name="collision_velocity_filter" output="screen">
+      <!-- remap incoming variables -->
+      <remap from="~obstacles" to="/local_costmap_node/costmap/obstacles" />
+      <remap from="~teleop_twist" to="/base/collision_velocity_filter/command" />
+      <!-- remap outgoing variables -->
+      <remap from="~command" to="/base/twist_mux/command_safe" />
+      <remap from="~relevant_obstacles" to="/base/collision_velocity_filter/relevant_obstacles" />
+      <!-- load parameter file -->
+      <rosparam file="$(arg pkg_hardware_config)/$(arg robot)/config/collision_velocity_filter_params.yaml" command="load" />
+  </node>
```
Comment (fmessmer) [inline cob_bringup/tools/base_collision_observer.launch]: indentation!
we consistently use tabs here...

File: cob_bringup/robots/raw3-5.xml (hunk 1 above)
Comment (destogl) [inline cob_bringup/robots/raw3-5.xml:30]: As fas as I checked I didn't find anything. raw3-5 is only with lms1xx

File: cob_bringup/tools/base_collision_observer.launch (hunk 2 above)
Comment (destogl) [inline cob_bringup/tools/base_collision_observer.launch]: Ok I'll change it.

**Comment by NadiaHG** [comment]
CR: OK

File: cob_bringup/drivers/sick_lms1xx.launch [hunk 3]
```diff
@@ -6,11 +6,11 @@
 	<arg name="name"/>
 
 	<!-- load parameter -->
-	<rosparam ns="$(arg name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
+	<rosparam ns="$(arg name)/driver" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
```
Comment (fmessmer) [inline cob_bringup/drivers/sick_lms1xx.launch]: why did you add `driver` to the namespace?
this would be different from topic names, the other robots are using...

Comment (floweisshardt) [inline cob_bringup/drivers/sick_lms1xx.launch]: that probably needs modification in the implementation of cob_driver/cob_sick_lms1xx package? @destogl?

Comment (destogl) [inline cob_bringup/drivers/sick_lms1xx.launch]: Because private names are used and node is starting with the name "driver" (see line 12 of this file). I wanted to make it more compatible to sick_lms300.

I am pretty sure that only raw3-5 is using lms1xx. So says the [care-o-bot web page](http://wiki.ros.org/Robots/Care-O-bot/distribution) and I greped the whole workspace.

Comment (fmessmer) [inline cob_bringup/drivers/sick_lms1xx.launch]: Yes, the difference is that lms1xx uses a private NodeHandle to retrieve the parameters while s300 doesn't! In order to make it consistent, I'd prefer to modify the cob_driver/sick_lms1xx to use a non-private NodeHandle, too! You can simply apply this change within your PR [here](https://github.com/ipa320/cob_driver/pull/255/files#diff-9136584c6b854bd371b524c4808d8c8fR60)

Comment (destogl) [inline cob_bringup/drivers/sick_lms1xx.launch]: Ok, will do!

**Comment by fmessmer** [comment]
Namespace adjustments has been done in the driver in https://github.com/iirob/cob_driver/commit/bd12ee497acede386db14479eea77b2e1cce83b9

I would like to see some "Tested on HW: OK" here

**Comment by destogl** [comment]
I mean, I tested on HW, but not sure if that counts :)

I always tray to commit only stuff that are working, especially if I am doing pull request. If it's something for testing I note it explicitly.

**Comment by destogl** [comment]
So if counts:

Tested on HW: OK

File: cob_bringup/robots/raw3-5.xml [hunk 4]
```diff
@@ -23,17 +23,35 @@
...
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="name" value="base_laser_rear" />
 		</include>
-		<include file="$(find cob_bringup)/drivers/relayboard.launch" >
+		<!--<include file="$(find cob_bringup)/drivers/relayboard.launch" >
 			<arg name="sim" value="true" />
+		</include>-->
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-5.xml]: Please remove relayboard sim
According to https://github.com/ipa320/cob_robots/issues/403#issuecomment-179844477, the functionality is taken over by phidgets

**Comment by fmessmer** [comment]
@destogl can you please address my last [comment](https://github.com/ipa320/cob_robots/pull/400#discussion_r51872616), then we will merge!

**Comment by destogl** [comment]
Done!

Sorry for not being active last days.

**Comment by fmessmer** [comment]
ok, I'm mergin this now...
